doc/source/dotnet.rst

```diff
@@ -15,7 +15,9 @@ These classes include PyObject, PyList, PyDict, PyTuple, etc.
 At a very high level, to embed Python in your application one will need
 to:
 
--  Reference ``Python.Runtime.dll`` (e.g. via a ``PackageReference``)
+-  Reference ``Python.Runtime.dll`` (e.g. via a ``PackageReference``,
+   the ``Runtime.PythonDLL`` property or the ``PYTHONNET_PYDLL``
```

Comment:
This is wrong. The `Runtime.PythonDLL` and `PYTHONNET_PYDLL` reference `libpython*.so/dylib` or `python*.dll`.

Reply:
Ah, I didn't know that there's a difference. But since it's now mandatory to reference a Python DLL, I think it's a good idea to mention it here. Maybe as a separate bullet point?